### PR TITLE
Add tutorial how to install webhook to trigger shell script

### PR DESCRIPTION
**Pull request by PatrickWe**
I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: (Patrick Weber, netcup_wiki@icloud.com)

### CODE REVIEW

File: community-tutorials/how-to-install-and-configure-webhook-to-trigger-a-shell-script/01-en.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Later in this tutorial we will execute our webhook with a http request.
+With the property "include-command-output-in-response" webhook will include the stdoutput of the shellscript to the http response body. With the property "include-command-output-in-response-on-error" webhook include the erroutput of the shellscript to the http response body. The http-request will wait until the full output is collected. If the exit-code of the shell-script is zero, the status-code of a http-request is 200. If the exit-code is not zero, the status-code is not 200.
+You have the option to remove the two "include-*" properties, then you will not see the output of the shellscript in the http-response body. For security reasons, its recommended to not include the output in the http-response-body.
+
+# Step 3 - Create the app-directory
+Create the app directory: ```mkdir -p /var/app```
+
+# Step 4 - Create the webhook-entrypoint.sh and make it executeable
+Create the file ```/var/app/webhook-entrypoint.sh``` with the content:
+
```
Comment (d-schaffer) [inline community-tutorials/how-to-install-and-configure-webhook-to-trigger-a-shell-script/01-en.md:58]: You describe how I create a folder. good

Then I have to write a content in a file that is in this folder.  
**But how do I create the file?**

**Review by d-schaffer — CHANGES_REQUESTED** [review]
need help

File: community-tutorials/how-to-install-and-configure-webhook-to-trigger-a-shell-script/01-en.md (hunk 1 above)
Comment (PatrickWe) [reply community-tutorials/how-to-install-and-configure-webhook-to-trigger-a-shell-script/01-en.md:70]: done, thanks a lot

**Comment by netcupClaudiaM** [comment]
Hi PatrickWe, thank you for contributing! It's been a while since you opened this PR and it probably is an open secret we're currently overwhelmed with reviewing all the awesome tutorials we got. Please be patient - we're about to publish tutorials this and next week. Thank you!

**Comment by PatrickWe** [comment]
Please delete all of my opened pull requests. It is outrageous, that vouchers are not allowed for more than the half of your products. Shame on you.

**Pull request closed without merging**